checks/dangerous_workflow.go

```diff
@@ -81,8 +81,10 @@ type triggerName string
 
```

Comment:
No where else to place this comment -- about the title:
- "add" rather than "added' (imperative rather than past tense. not sure if mandatory, but that's the Linux kernel guideline so I'd assume it'd be preferred).
- as "a" dangerous trigger